src/app/components/footer/footer.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss'],
+})
+export class FooterComponent implements OnInit {
+  ngOnInit(): void {
```

Comment:
Why do you need implements OnInit on footerComponent?

Reply:
fix code